snapshot-labs/sx-monorepo
# feat: Votes page horizontal scrollable on mobile

### Summary

Closes: #331 

- Adds a horizontal scroll for votes page on mobile
- Hide right side bar (results and choices) on mobile (sticky needs a fixed height)
- Remove padding at bottom for votes page (sticky needs a fixed height)
- On swipe, it should not open right sidebar https://github.com/snapshot-labs/sx-monorepo/pull/618#discussion_r1716404917

### How to test

1. Go to votes page in any proposal https://bafybeihxvycavj4u7au7kmcwgwyp5islxrzj562hoskasnyieqgkncpbma.on.fleek.co/#/s:safe.eth/proposal/0x8d97075eabfdf3fb9f43f3b490e8e5a7326d77bee919b78543621e36e5b2d233/votes
2. Everything should work same as before on desktop
3. On mobile, you should see a horizontal scroll
